client/app/_consts/iconColor.ts

```diff
@@ -37,3 +37,15 @@ export const iconColor = [
     value: 'WHITE',
   },
 ] as const
+
+export const colorMap = {
+  RED: 'text-red-500',
```

Comment:
If we choose this colour, it will be very similar to the colour of the delete button. So I think pink-700 or something is better? (It is not technically red but it should be fine)

Reply:
fix it! 
8b448ff822baa353a3e0b55f56b2897a2ba4f855